arbnode/sequencer.go

```diff
@@ -327,13 +329,13 @@ func (s *Sequencer) ctxWithQueueTimeout(inctx context.Context) (context.Context,
 	return context.WithTimeout(inctx, timeout)
 }
 
-func (s *Sequencer) PublishTransaction(parentCtx context.Context, tx *types.Transaction) error {
+func (s *Sequencer) PublishTransaction(parentCtx context.Context, tx *types.Transaction, options *arbitrum_types.ConditionalOptions) error {
 	sequencerBacklogGauge.Inc(1)
```

Comment:
I've added conditional tx rejected/accepted metrics on each stage:
* rejected/accepted by pre-checker based on current state
* rejected/accepted by pre-checker based on old state
* rejected/accepted by sequencer

We could in theory count also all executed storage checks (slot-value + storage root checks), but it will need to be added in geth submodule. Do you think it would be a valuable metric?